contracts/src/systems/combat/tests/battle_leave_test.cairo

```diff
@@ -94,14 +94,13 @@ fn set_configurations(ref world: WorldStorage) {
         .write_model_test(
             @SettlementConfig {
                 config_id: WORLD_CONFIG_ID,
-                radius: 50,
-                angle_scaled: 0,
                 center: 2147483646,
-                min_distance: 1,
-                max_distance: 5,
-                min_scaling_factor_scaled: 1844674407370955161,
-                min_angle_increase: 30,
-                max_angle_increase: 100,
+                base_distance: 10,
+                min_first_layer_distance: 30,
+                points_placed: 0,
+                current_layer: 1,
+                current_side: 1,
+                current_point_on_side: 0,
```

Comment:
Could you write a default config and put that in the testing default config? It'll be clearer to have a function that returns it :)